frontend/src/css/inbox.css

```diff
@@ -19,3 +19,11 @@ styles that cannot be added inside a const
   -ms-grid-column: 2;
   grid-column: 2;
 }
+
+.email-preview-button:focus {
+  background-color: #c6ecef !important;
```

Comment:
Anytime you use important, can you add a comment explaining why it's necessary?

Reply:
Comment added in commit #https://github.com/code-for-canada/project-thundercat/pull/201/commits/f4f8a14ad766d538b60b9b444f2124f6a1abd87b